### PR TITLE
Document semver behavior

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: content/vendor/packaging/promoting-releases.md [hunk 1]
```diff
@@ -30,3 +30,16 @@ Additionally, each promoted release must be given a required status (required or
 - License values are synced with the values set in the vendor portal when the customer syncs the license.
 - Releases will not be editable after being promoted to a channel.
 - Release notes, version numbers, and the required status may be edited after promotion by visiting the channel’s history.
+
+## Semantic Versioning
+
+Semantic versioning is available in KOTS v1.58.0 and later.
+
+You can use the semantic version format for the version label that you assign to a release in the admin console. For more information about semantic versioning, see [Semantic Versioning 2.0.0](https://semver.org).
```
Comment (sgalsaleh) [inline content/vendor/packaging/promoting-releases.md:38]:
```suggestion
You can use the semantic version format for the version label that you assign to a release in the vendor portal. For more information about semantic versioning, see [Semantic Versioning 2.0.0](https://semver.org). 
```

File: content/vendor/packaging/promoting-releases.md [hunk 2]
```diff
@@ -30,3 +30,16 @@ Additionally, each promoted release must be given a required status (required or
...
+## Semantic Versioning
+
+Semantic versioning is available in KOTS v1.58.0 and later.
+
+You can use the semantic version format for the version label that you assign to a release in the admin console. For more information about semantic versioning, see [Semantic Versioning 2.0.0](https://semver.org). 
+In order for this label to be used to sequence releases on a channel, semantic versioning must be enabled by turning on the `Require semver for releases promoted to this channel` toggle in the channel settings.
+If you use semantic versioning on a channel, we recommend that you always use semantic versioning on that channel.
+This toggle is disabled by default.
+
+If the version label that you assign to a channel is in the semantic version format, the admin console uses the version label for sequencing of releases during updates.
```
Comment (sgalsaleh) [inline content/vendor/packaging/promoting-releases.md:43]:
```suggestion
If the version label that you assign to a channel is in the semantic version format, the KOTS admin console uses the version label for sequencing of releases during updates.
```

File: content/vendor/packaging/promoting-releases.md [hunk 3]
```diff
@@ -30,3 +30,16 @@ Additionally, each promoted release must be given a required status (required or
...
+For releases that do not use semantic versioning, the admin console sequences them in the order of their creation.
```
Comment (sgalsaleh) [inline content/vendor/packaging/promoting-releases.md:44]:
```suggestion
For releases that do not use semantic versioning, the KOTS admin console sequences them in the order of their creation.
```

Comment (sgalsaleh) [inline content/vendor/packaging/promoting-releases.md:44]:
```suggestion
For releases that do not use semantic versioning, the admin console sequences them in the order of their promotion to the channel.
```

**Comment by paigecalvert** [comment]
Hi @sgalsaleh, thank you for adding these corrections. One thing I wanted to ask about before merging:

I see you changed "admin console" to "vendor portal". There is a second mention of "admin console"  here:

>For releases that do not use semantic versioning, the admin console sequences them in the order of their promotion to the channel.

Should this also be changed to "vendor portal"?

**Comment by sgalsaleh** [comment]
@paigecalvert nope, just the one i suggested.

**Pull request merged**